test/e2e/README.md

```diff
@@ -0,0 +1,95 @@
+# E2E Test Instruction
```

Comment:
There is document related to e2e, might be good to remove it and add a link pointing to this one